pkgs/servers/photoprism/libtensorflow.nix

```diff
@@ -80,7 +80,10 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
+    # until nixos search supports meta.repository, these attributes will stay
+    # duplicated.
     homepage = "https://dl.photoprism.app/tensorflow/";
+    repository = "https://dl.photoprism.app/tensorflow/";
```

Comment:
does not seem required. i am not sure how to reach this attribute through photoprism